### PR TITLE
Bump versions pre 0.2.2

### PR DESCRIPTION
**Pull request by hratoanina**

### CODE REVIEW

**Review by muursh — APPROVED** [review]

File: field/Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "plonky2_field"
 description = "Finite field arithmetic"
-version = "0.2.0"
+version = "0.2.1"
```
Comment (Nashtare) [inline field/Cargo.toml:4]: Why is this bumped?

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,8 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## Unreleased
+## [Unreleased]
 
+## [0.2.2] - 2024-03-21
```
Comment (Nashtare) [inline CHANGELOG.md:10]: Unless I'm mistaken, this is only for starky, so the CHANGELOG should reflect it for clarity.

File: starky/Cargo.toml [hunk 3]
```diff
@@ -26,7 +26,7 @@ log = { workspace = true }
 num-bigint = { version = "0.4.3", default-features = false }
 
 # Local dependencies
-plonky2 = { version = "0.2.0", path = "../plonky2", default-features = false }
+plonky2 = { version = "0.2.1", path = "../plonky2", default-features = false }
```
Comment (Nashtare) [inline starky/Cargo.toml:29]: Why the bump?

File: starky/Cargo.toml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "starky"
 description = "Implementation of STARKs"
-version = "0.2.1"
+version = "0.3.0"
```
Comment (Nashtare) [inline starky/Cargo.toml:4]: The CHANGELOG says 0.2.2 but this is bumped to 0.3.0.

**Review by Nashtare — COMMENTED** [review]
@muursh can you make sure the comments I made are relevant / applied if necessary before proceeding to a new release?

**Comment by Nashtare** [comment]
Ah I just noticed the new versions have already been published. I guess it's fine but given all (but starky) are basically just copies of the previous ones without any changes, we should **_really_** not have to bump crates needlessly in the future.

**Pull request merged**